### ISSUE TITLE
Filters applied to non-window screenshots

### ISSUE THREAD

**Issue by lenemter** [elementary/gala#1680]
### What Happened?

When screenshoting a window with enabled filters, screenshots doesn't reflect the filter.

### Steps to Reproduce

1. Turn on any filter (e.g. grayscale)
2. Screenshot a window

### Expected Behavior

Screenshot with a filter.

### OS Version

7.x (Horus)

### Software Version

Latest release (I have run all updates)

**Comment by danirabbit** [comment]
I don’t think this is a bug. I think it’s a feature. I’m not sure there’s really value in having these filters applied to a screenshot since it’s your personal accommodation

**Comment by lenemter** [comment]
> I don’t think this is a bug. I think it’s a feature. I’m not sure there’s really value in having these filters applied to a screenshot since it’s your personal accommodation

But filters are applied to screen screenshots currently. Should we disable this behaviour then?

**Comment by tintou** [comment]
The filter shouldn't be applied to the screenshot content, otherwise:
 * Other people will see that you have a filter enabled if you share the image, and that's none of their business
 * We end-up with the filter being applied twice if we then view the screenshot with the same filter
